### PR TITLE
WSTEAM1-969: Add inline link styling

### PR DESCRIPTION
**Pull request by shayneahchoon**
Resolves [WSTEAM1-969](https://jira.dev.bbc.co.uk/browse/WSTEAM1-969)

Overall changes
======
Adds inline link styling: 

Screenshot 2024-05-20 at 12 00 47

Testing
======
1. _List the steps used to test this PR._

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: ws-nextjs-app/pages/[service]/send/[id]/styles.ts [hunk 1]
```diff
@@ -32,14 +32,27 @@ export default {
...
-
       [mq.GROUP_4_MIN_WIDTH]: {
         gridColumn: '1 / span 8',
         paddingBottom: `${spacings.QUADRUPLE}rem`,
       },
     }),
   mainContent: ({ spacings, palette, mq }: Theme) =>
     css({
+      a: {
+        fontWeight: 700,
```
Comment (karinathomasbbc) [inline ws-nextjs-app/pages/[service]/send/[id]/styles.ts]: Is `fontWeight` usually something we hardcode like this? I thought this value was computed based on the `fontVariant`? e.g. https://github.com/bbc/simorgh/blob/f5233e91760bbac4c0a87d516cfe4fb9f59c6835/src/app/components/Curation/Subhead/index.styles.ts#L6 - if you provide `sansBold` as the fontVariant, then it should send the correct fontWeight based on the service and its font

**Review by karinathomasbbc — APPROVED** [review]

**Comment by amoore108** [comment]
Is it worth following the pattern in https://github.com/bbc/simorgh/pull/11631/ ? It re-uses the inline link styling to the `description` and `privacyNotice` elements directly, so maybe we just need to add in the font weight changes to these 2 classes?

File: ws-nextjs-app/pages/[service]/send/[id]/styles.ts [hunk 2]
```diff
@@ -37,22 +36,9 @@ export default {
...
-        '&:visited': {
-          color: palette.METAL,
-          borderBottom: `${pixelsToRem(1)}rem solid ${palette.METAL}`,
-        },
-        '&:focus, &:hover': {
-          borderBottom: `${pixelsToRem(2)}rem solid ${palette.POSTBOX}`,
-          color: palette.POSTBOX,
-        },
-      },
+      a: getInlineLinkStyles(palette),
```
Comment (amoore108) [inline ws-nextjs-app/pages/[service]/send/[id]/styles.ts]: I'm not sure if we need this here as the `description` and `privacyNotice` classes already define this CSS: https://github.com/bbc/simorgh/blob/latest/ws-nextjs-app/pages/%5Bservice%5D/send/%5Bid%5D/Form/styles.ts#L20-L41

**Review by amoore108 — APPROVED** [review]
Nice!

**Pull request merged**